src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar-style {
+  position: relative;
+  margin-top: 20px;
+}
+
+.btn-img {
+  position: absolute;
+  border: none;
+  background-color: transparent;
+  padding: 0;
+}
+
+.btn-img-big {
+  top: 25px;
+  left: 26px;
+}
+
+.search-img {
+  width: 100%;
+  height: auto;
+  display: block;
+}
+
+.search-input {
+  font-family: inherit;
+  font-weight: 700;
+  color: #3d4e61;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-input-big {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.btn-img-small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-input-small {
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+}
```

Comment:
you should have default search-bar and big/small. Not big and small at the same time. One of them is just search-bar without big/small word.